packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -61,8 +61,12 @@ export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JS
     ]
   );
 
+  // gap space uses pixels
   return (
     <ContextualMenu
+      alignTargetEdge={true}
```

Comment:
Please add the comment you have in PR (about flyout flyin' away) above line 67, so that next time we see this we know why.